app/src/main/res/values/strings.xml

```diff
@@ -468,6 +468,7 @@
     <string name="behavior_when_minimized">Behavior when minimized</string>
     <string name="external_player">External player</string>
     <string name="screenshot">Screenshot</string>
+    <string name="screenshot_is_saved">Screenshot is saved</string>
```

Comment:
This sounds a bit unnatural, maybe "Screenshot has been saved" or just "Screenshot saved"?